pytest_tldr.py

```diff
@@ -21,7 +21,7 @@ class ExitCode:
         TESTS_FAILED = EXIT_TESTSFAILED
 
 
-__version__ = '0.2.2'
+__version__ = '0.2.3b'
```

Comment:
The version attribute is not something a PR should be editing - it's a project management concern.

Reply:
gotcha, sorry about that!